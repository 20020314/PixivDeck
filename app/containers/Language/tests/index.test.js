// @flow
import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import { Language } from '../index'

test('snapshot', () => {
  const children = <h1>Test</h1>
  const wrap = shallow(
    <Language locale="en" messages={{ en: {}, ja: {} }}>
      {children}
    </Language>
  )
  expect(toJSON(wrap)).toMatchSnapshot()
})
