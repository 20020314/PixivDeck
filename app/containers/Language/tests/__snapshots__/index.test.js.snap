// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<IntlProvider
  defaultLocale="ja"
  locale="en"
  messages={Object {}}
>
  <h1>
    Test
  </h1>
</IntlProvider>
`;
